problem_builder/tests/integration/xml_templates/step_builder.xml

```diff
@@ -64,7 +64,7 @@
     <sb-conditional-message score_condition="perfect">Great job!</sb-conditional-message>
     <sb-conditional-message num_attempts_condition="cannot_try_again">This message is shown when you run out of attempts.</sb-conditional-message>
     <sb-review-score/>
-    {% if include_review_tips %}
+    {% if include_review_tips_block %}
     <sb-review-per-question-feedback/>
```

Comment:
@bradenmacdonald Nit: Since you touched the line above, you could fix indentation here.